src/docs/sphinx/advancedExamples/validationStudies/pennyFracViscosityDominated/pennyFracViscosityDominatedFigure.py

```diff
@@ -0,0 +1,143 @@
+import numpy as np
```

Comment:
looks like you didn't use the existing [HydrofractureSolutions.py](https://github.com/GEOSX/GEOSX/blob/develop/scripts/postProcessing/hydraulicFractureVerification/HydrofractureSolutions.py)

Was there a problem using this?

Reply:
While testing this case, I coded up the python script separately, but it should be the same as the existing one.

Reply:
The point is that we have a single place where all the analytical solutions for KGD and Penny problems are contained rather than having it duplicated in multiple scripts. Could you use the existing capability?

Reply:
Use existing python script and update the examples for both KGD and Penny fractures